client/src/actions/authActions.js

```diff
@@ -1,4 +1,4 @@
-import { USER_LOADED, USER_LOADING, LOGIN_SUCCESS, AUTH_ERROR, LOGIN_FAIL, REGISTER_SUCCESS, REGISTER_FAIL, LOGOUT_SUCCESS, CLEAR_SUBJECTS, DELETING_USER, USER_DELETED,UPDATING_USER,USER_UPDATED } from '../actions/actionTypes';
+import { USER_LOADED, USER_LOADING, LOGIN_SUCCESS, AUTH_ERROR, LOGIN_FAIL, REGISTER_SUCCESS, REGISTER_FAIL, LOGOUT_SUCCESS, CLEAR_SUBJECTS, DELETING_USER, USER_DELETED,UPDATING_USER,USER_UPDATED, SET_TABLE_PAGE } from '../actions/actionTypes';
```

Comment:
create another file for this action since its already been a large file and moreover it has all the API calls integrated to it.